.github/workflows/ensure-labels.yaml

```diff
@@ -1,18 +1,16 @@
-name: Ensure labels
+name: Apply labels in .github/labels.yaml
 on:
   push:
-    branches:
-      - master
```

Comment:
As Pierre pointed out the other day, this workflow syncs the repository labels from what is defined in the `.github/labels.yaml` file. If we were to run this from the v1 branch, we would delete the `backport-v1` label each time we had a PR triggering this workflow in the `v1` branch. This was my mistake and we should revert it.

Reply:
In fact, we should even delete this workflow from the v1 branch.

Reply:
I'm a bit confused by what we want now, I'll let @pierreprinetti comment; you seem to have a stronger opinion on what we should be doing wrt these workflows in v1 branch. Let me know if you need help with that PR.

Reply:
Closing in favor of https://github.com/gophercloud/gophercloud/pull/2683